Drummerboy444/Danki2
# Collision sounds

To test:
Stick in scene (Trees are good) and update the intances' colliders with the various physic materials.

Add CollisionSoundLevel.Low for the new optional parameter in sweeping-strike's (good to use sweeping strike as it's silent) call to Ability.TemplateCollision.

There are couple of things that still need to be done to complete the feature:
- Add physic materials to relevant prefabs (and add this to the prefab creation guide) (maybe even add some kind of runtime check to ensure we are setting these. Perhaps all prop-tags with a collider should have a physic material set?)
- Use collision sounds in abilities
- Exclude the caster from collision sounds (doesn't affect us yet because the player doesn't have a physic material)
- Make use of position in the collision sound event plays
